### DISCUSSION TITLE
Calling `FitAddon.fit()` after `dispose` throws an error

### DISCUSSION THREAD

**Discussion by Nokel81** [sourcelair/xterm.js#4739] [Q&A] [answered]
## Details
- `xterm-addon-fit`: `"0.5.0"`

### Steps to reproduce

```ts
const fitAddon = new FitAddon()
const xterm = new XTerm();

xterm.loadAddon(fitAddon);

xterm.dispose();
fitAddon.dispose();

fitAddon.fit(); // from a debounce
```

**Comment by Tyriar** [comment, marked as answer]
This is by design, you cannot use an object after dispose.

**Reply by Nokel81** [reply]
Oh okay, I'll work around it then. Looking at the code I would have assumed that dispose is the inverse of attach.
